aws/nextflow-launch-templates.tf

```diff
@@ -29,3 +29,20 @@ resource "aws_launch_template" "nf_lt_bigdisk" {
   }
   update_default_version = true
 }
+
+resource "aws_launch_template" "nf_lt_auto_scaled_ebs" {
+  name = "nextflow-launchtemplate-auto-scaled-ebs"
+  tags = var.default_tags
+  image_id = "ami-01c08d4de548477df"  # ccdl-nextflow-base-v2.0 image
```

Comment:
I see that you changed this back, which may be fine, though I worry it might be the cause of the issues I encountered. I realized that I needed to change the location of the aws cli in nextflow settings, which I had neglected to do, and might be having some effect, though I don't think this is the main problem. I worry some about having two version of the aws cli, one of which is presumably older and may be missing features?

https://github.com/AlexsLemonade/scpca-nf/blob/ef585fd1ab13ef24920a39579e5fb61c2c7a263c/config/profile_awsbatch.config#L5

One note is that according to the nextflow docs, the aws cli is mounted into the docker image in kind of a strange way, which could also cause trouble: https://www.nextflow.io/docs/latest/aws.html#aws-cli

Reply:
Keeping the old one so far. Filed https://github.com/AlexsLemonade/alsf-scpca/issues/169